plugins/auth/src/auth-oauth2/index.js

```diff
@@ -4,17 +4,15 @@ import { Helpers } from '../helpers';
 export default class oAuth2ImplicitGrant {
   constructor(settings = {}) {
     const defaultSettings = {
+      accessTokenExpiringNotificationTimeInSeconds: 60,
```

Comment:
Was it necessary to rename this property as well? If not, needs to be clarified with Philipp, because of breaking change.

Reply:
You're right - let me fix it